lib/scooter/httpdispatchers/classifier.rb

```diff
@@ -291,6 +291,9 @@ def update_node_group_with_node_group_model(node_group_model, update_hash)
           node_group_model.delete('deleted') if node_group_model['deleted'] == {}
         end
 
+        node_group_model.delete('serial_number')
```

Comment:
I think Anna introduced changes in the [v1 file](https://github.com/puppetlabs/scooter/blob/master/lib/scooter/httpdispatchers/classifier/v1/v1.rb#L39) instead of here in the classifier file; does it make sense to introduce those changes there? I don't really know how the `serial_number` and `last_edited` are used, but maybe it makes sense to catch and delete those keys in that v1 file?